scripts/variantstore/wdl/GvsJointVariantCallsetCost.wdl

```diff
@@ -0,0 +1,113 @@
+version 1.0
+
+import "GvsCallsetCost.wdl" as GvsCallsetCost
+
+workflow GvsJointVariantCallsetCost {
+    input {
+        String project_id
+        String dataset_name
+        String workspace_namespace
+        String workspace_name
+        String call_set_identifier
+    }
+
+    # the call_set_identifier string is used to name many different things throughout this workflow (BQ tables, vcfs etc),
+    # and so make sure nothing is broken by creative users, we replace spaces and underscores with hyphens
+
+    if(false) {
+      Array[String] excluded_submission_ids = []
+    }
+    File gatk_override = "gs://gvs_quickstart_storage/jars/gatk-package-4.2.0.0-552-g0f9780a-SNAPSHOT-local.jar"
+
+    ## TB : GB; 1 : 1024
+
+    Float write_API_cost = 0.02684353333 ## BigQuery Storage Write API: $0.025 per 1 GB. The first 2 TB per month are free.
```

Comment:
probably good to link to doc here, Google is the authority and prices may change

Reply:
per mobbing discussion changing to the same round numbers in the docs